packages/plugins/opentelemetry/src/spans.ts

```diff
@@ -127,10 +141,35 @@ export function setParamsAttributes(input: {
   );
 }
 
-export function setExecutionAttributesOnOperationSpan(
-  ctx: Context,
-  args: ExecutionArgs,
-) {
+export type OperationHashingFn = (input: {
+  document: DocumentNode;
+  operationName?: string | null;
+  variableValues?: Record<string, unknown> | null;
+  schema: GraphQLSchema;
+}) => string | null;
+
+const typeInfos = new WeakMap<GraphQLSchema, TypeInfo>();
+export const defaultOperationHashingFn: OperationHashingFn = (input) => {
+  if (!typeInfos.has(input.schema)) {
+    typeInfos.set(input.schema, new TypeInfo(input.schema));
+  }
+  const typeInfo = typeInfos.get(input.schema);
+
+  return hashOperation({
+    documentNode: input.document,
+    operationName: input.operationName ?? null,
+    schema: input.schema,
+    variables: input.variableValues ?? null,
```

Comment:
Using the variables as part of the hash is opt-in:

https://github.com/graphql-hive/console/blob/92756d8d0c5e13333229290a81f7cc3189a9df80/packages/libraries/core/src/client/types.ts#L132-L137

Reply:
```suggestion
    variables: null,
```